main.py

```diff
@@ -1,17 +1,22 @@
-import boto3
 import argparse
 import logging
 
 from AwsResources import AwsResources
+from Ssh import Ssh
+from rsync import perform_rsync
 
 parser = argparse.ArgumentParser(description='Load data on an AWS EBS volume')
+parser.add_argument('source', metavar='src', type=str, help='Your source directory')
+parser.add_argument('--dst', help='Specify a directory as destination. If None, src will be saved '
+                                  'in the root directory')
 parser.add_argument('--existing-volume', help='specifies an already existing EBS volume by its '
```

Comment:
Isn't it possible to automatically check if the volume already exists by name? Is the volume name a unique identifier?

Reply:
No, it's not, you can have multiple EBS with the same name, in the end a name is just an AWS tag, you can also don't have it